core/common/src/main/java/org/sopt/common/extension/CommonExtension.kt

```diff
@@ -0,0 +1,8 @@
+package org.sopt.common.extension
+
+import android.view.View
+import com.google.android.material.snackbar.Snackbar
+
+fun showSnack(anchorView: View, message: () -> String) {
+    Snackbar.make(anchorView, message(), Snackbar.LENGTH_SHORT).show()
+}
```

Comment:
```suggestion
fun View.showSnack(message: () -> String) {
    Snackbar.make(anchorView, message(), Snackbar.LENGTH_SHORT).show()
}
```

View의 확장함수로 선언할 수도 있을 것 같습니다?!

Reply:
하지만... binding.root.showSnack은 너무 안이뻐요!!